notebooks/official/model_evaluation/custom_tabular_regression_model_evaluation.ipynb

```diff
@@ -0,0 +1,2224 @@
+{
```

Comment:
<p>How is this section related to the goal of this notebook? I think it looks like extra stuff unless there is a reason for mentioning this. If not, please remove because it's a distraction.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/942/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='67'/>

Reply:
<p><strong>Please find this from docs</strong>:</p><p>To use Vertex Explainable AI with a custom-trained model, you must configure certain options when you create the&nbsp;<code>Model</code>&nbsp;resource that you plan to request explanations from,</p><p> </p><p>If you want to use Vertex Explainable AI with an AutoML tabular model, then you don't need to perform any configuration; Vertex AI automatically configures the model for Vertex Explainable AI.&nbsp;</p>

Reply:
<p>This section is required for configuring explanations to model</p><p> </p><p> </p>

Reply:
<p>https://cloud.google.com/vertex-ai/docs/explainable-ai/configuring-explanations-feature-based</p>

Reply:
Sounds good!

Reply:
<p>Do you need to explain this all here, or can you just link to the documentation and say "learn more about explanation metadata here"</p><p> </p><p>This is a lot of detail for a feature that isn't relevant to the main goal of the notebook, as Soheila said. </p>

Reply:
<p>removed text and added link</p>